storm-core/src/jvm/backtype/storm/Config.java

```diff
@@ -1087,6 +1087,17 @@
     public static final String TOPOLOGY_METRICS_CONSUMER_REGISTER = "topology.metrics.consumer.register";
     public static final Object TOPOLOGY_METRICS_CONSUMER_REGISTER_SCHEMA = ConfigValidation.MapsValidator;
 
+    /**
+     * A map of metric name to class name implementing IMetric that will be created once per worker JVM
+     */
+    public static final String TOPOLOGY_WORKER_METRICS = "topology.worker.metrics";
+    public static final Object TOPOLOGY_WORKER_METRICS_SCHEMA = Map.class;
+
+    /**
```

Comment:
It would be better to explain the intention from source code comment or configuration file, or doc, since users can't know that easily.